modules/netweaver/main.tf

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = "~> 0.11.0"
+}
+
+locals {
+  device_name_1 = "${var.instance_name}-${var.device_1}"
+  device_name_2 = "${var.instance_name}-${var.device_2}"
+  device_name_3 = "${var.instance_name}-${var.device_3}"
+
+  access_config = {
+    "0" = []
```

Comment:
We should replace `"0"` and `"1"` with `false` and `true` to make this more obvious.

Reply:
added false for "0" and true for "1", will need to test and see the module works with these changes.